src/app/component/Burger/build_controls.cljs

```diff
@@ -8,29 +8,25 @@
   (action [{:keys [state]}]
     (let [id (:id props)
           price (:price props)]
-      ;(js/console.log id price (get-in @state [:order/id 1 :order/total-price]))
       (swap! state update-in [:order/id 1 :order/ingredients id :count] inc)
       (swap! state update-in [:order/id 1 :order/total-price] + price))))
 
 (defmutation remove-ingredient [props]
   (action [{:keys [state]}]
      (let [id (:id props)
            price (:price props)]
-       ;(js/console.log id price (get-in @state [:order/id 1 :order/total-price]))
        (swap! state update-in [:order/id 1 :order/ingredients id :count] dec)
        (swap! state update-in [:order/id 1 :order/total-price] - price))))
 
 
 (defmutation toggle-modal [_]
   (action [{:keys [state]}]
-    (let [modal? (get-in @state [:ui/id 1 :ui/modal?])]
-      (js/console.log "Toggle Modal: "  modal?)
-      (swap! state assoc-in [:ui/id 1 :ui/modal?] (not modal?)))))
+    (let [modal? (get-in @state [:singleton :app.ui/modal :ui/modal])]
```

Comment:
Ik kan er niet op commenten maar in deze namespace staat `Burger`.. ik zie nergen in clojure mensen CamelCase namespace names gebruiken

Reply:
oops.... vergeten te renamen.